test/e2e_node/e2e_remote.go

```diff
@@ -82,12 +82,14 @@ func GetHostnameOrIp(hostname string) string {
 // the binaries k8s required for node e2e tests
 func CreateTestArchive() (string, error) {
```

Comment:
The first `getArchive` is only used to trigger the build, and the following calls are used to get build result. And the following ones will check the error.